README.md

```diff
@@ -21,17 +21,17 @@ More details and a documented process can be found in
 
 During the research I've identified the following results: in terms of efficiency
 
-* `QWERTY` - `0%` (base line)
+* `QWERTY` - `0%` (baseline)
 * `Dvorak` - `+77%`
 * `Colemak` - `+84%`
 * `Workman` - `+101%`
 * `Halmak` - `+134%`
 
-Please refer to [this article](http://nikolay.rocks/2016-12-20-the-halmak-reborn)
+Please refer to the article *[Halmak Reborn]*
 for the more detailed breakdown of the results.
 
 All scripts are opensourced and can be verified in the
-[keyboard-gentics](https://github.com/MadRabbit/keyboard-genetics) repository.
+[keyboard-genetics] repository.
```

Comment:
Why all these newlines? This is markdown.

Reply:
This is indeed markdown.

Markdown lets reduce the number of columns that a paragraph. A single newline will be ignored by the parser.

@MadRabbit seemed to be using this feature to begin with, so I followed suit to keep the style of the codebase unified.